src/components/Collapse.jsx

```diff
@@ -15,7 +15,7 @@ const Collapse = ({
   onMarkerDragEnd,
 }) => {
   return (
-    <React.Fragment>
+    <div style={{ textAlign: "center" }}>
```

Comment:
Any reason for setting this inline and not using a class?

Reply:
@tmccoy529 i believe this can be changed to `text-center` class using https://getbootstrap.com/docs/4.0/utilities/text/

Reply:
done